src/App.jsx

```diff
@@ -14,45 +17,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function getPrepareGoods(goods, sortType, isReverse) {
+  let prepareGoods = goods;
+
+  if (!sortType && !isReverse) {
+    return prepareGoods;
+  }
+
+  prepareGoods = [...goods].sort((good1, good2) => {
+    switch (sortType) {
+      case 'alphabetically':
+        return (good1.localeCompare(good2));
+
+      case 'length':
+        return (good1.length - good2.length);
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
let's add the if to check if we need to sort the array

Reply:
Thanks, but I think that I misunderstood the last comment